threescale_api/defaults.py

```diff
@@ -314,6 +314,15 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return str(self)
 
+    def __eq__(self, other) -> bool:
+        return (
+            self.__class__ == other.__class__ and
+            (
+                self.entity_name == other.entity_name or
```

Comment:
do all the objects have entity name? (otherwise traceback will go from this

Reply:
You are right, I've forgotten comparing with objects which are not under DEfaultResource